clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -2017,6 +2016,18 @@ void handleFailure(Throwable throwable) {
         return new DescribeConfigsResult(allFutures);
     }
 
+    private Config describeConfigResult(DescribeConfigsResponseData.DescribeConfigsResult describeConfigsResult) {
```

Comment:
It looks like we are not calling `configSynonyms()` anymore, so we can remove that private method